Event/UserEvent.php

```diff
@@ -20,7 +20,7 @@ class UserEvent extends Event
     private $request;
     private $user;
 
-    public function __construct(UserInterface $user, Request $request)
+    public function __construct(UserInterface $user, Request $request = null)
```

Comment:
Because UserManipulator::getRequest() private method can return null. The case is when you are using SF <2.4 and request scope is not active.